.github/workflows/shark_model_suite.yml

```diff
@@ -0,0 +1,106 @@
+# Runs the shark model suite on iree wheels from build package workflow
+name: Validate Shark Model Suite
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_id:
+        description: 'Release id to publish'
+        required: true
+      package_version:
+        description: 'Version of the package'
+        required: true
+      build_run_id:
+        description: 'Run ID for the build_package.yml workflow that triggered this workflow'
+        required: true
+
+jobs:
+  build-validate:
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [a100, MacStudio, ubuntu-latest]
```

Comment:
I think these have to match the runners in the issue tracker. MacStudio wont be available :D